010_spring_system/spring.ipynb

```diff
@@ -0,0 +1,525 @@
GExDDubjuVR4ehSB/cuEnM0jJdrhem9Z/fr4V/IylRTNO6I46AGOPKaiPiTZDg5i2hWLawvkLRRGDZChVzdH5Djvj+nK2il0bXDd9NXUIATjTPgL6VouO2hrtnd+vf9dBti95Gpd9NkOI3P5Di18dI4ePPS8nn30gwZ6O4O3eU5FOO0/5Vrqy2YqS3EkkLiVZmaak6mHC7LRAf6EFfpHbW+raBQy6qIwQJIYQQUiMUkQghhBBCCCGEkBjGldFajJYt7CURM79QpLjEXqo7Cd27ai+itJuvlqRDDlCBIq5wucRo1UpcaanqpDF9PhH0Booygnn5GicY3JIXWl6zToIbcsTwesW7zwhJOvZwcXfqoI+ZCQkibk+dBD/sJ96Re6gjC/im/CGFT7wgBQ8/IyXfTxCzqEg8u/SR5HNOkaTTTih3gMH15Eqze1dZfwduKLMO+1e4uAlURLKFMUIIIYTEDhSRCCGEEEIIIYSQGMZISREjuaK/kFlWJqav/k4kOE4SBg+QhN0GiZGZbt8bX7jbtRFXeroYCR4Rr9e6I/rKIsF16yWwJlvEH+aS8rjVgZRy3mni3WdPMdq2EUm03n9evpR8+a0UPPo/KXrpTSn9+nvtaeWfO796HKFhqIiUfPJx4tm5l7WjBLXHEZxrEAyTDjtIUq+5VFLOPEmj7xxc6LWEdWWjbqQ6ON3Qf8qJzwOBFaEIQZN9kQghhJCYwn23hT1PCCGEEEIIIYSQWMM0xffzZPH9OUsXIQJ599xNjNRUXSZhwLmTnCievn0k6YC9tfdPtIFIwrLfpmlfJMV6z4kjhknqJWerE8nVuqX1JJHgytUSWLNWAkuWi2/y7+L7/S/xzZqj+4H/73niatdWhZxwEIfnsT6zu2tnja3z9Owm3j2GSOLhoyT5VPRAGl7J1QYgMvkm/W79nWW6nDCwn/Vvhpb3SdoaiM3z473M/FsdX2ZJqSQM6i/eIQM17o4QQgghsQFFJEIIIYQQQgghJIYxrKl0wq8VItKwIZKw527ioohUDVdmunj67SyJe+0hbrhtXNHnRHIlJ6ngIgG/eDp1kMR9R0jK2aeI13rPcJ3BUYSYOHdWGzHQxyoYFMPnE7OgUMyNm8XMz1dh0Tt0YHnvpHAQT4fYOu/u1n5i7SuJ+++t4pG7QzsRt9t+VgXmps1SNnGqBBYt1WXPgF3EO3x3cbVupctbw/B4xL9wifhnzhGzsEjvg2iVMLB/SAgjhBBCSEzAODtCCCGEEEIIISSGMQ1DDDhsHEEk4FfnB6kBa13BQWPARVODYBIVJCVJ4mEHSdpNV0kqpusuUweSCkg2RkqyJB58gKRef7m0sJ6TdsMV+jxMaTdcKanXXiqewbvaz64ZIy1VeytBWENc3lZBL6lwsQ39pMy67V/oq2S0y7KXRPwrV0tw/QZ7iRBCCCGxAEUkQgghhBBCCCEkhjHcbhUVDLtvjVlUEnKykJgFwk7CkIHiHTFMY+dqFHms+9DbKOm4IyTligsk7dZrJO2W0ZJyyTmSdOQhKhBFBJdLTNjdwqiyuFXcXTqLu0OFiBRYvkoCa7PtJUIIIYTEAhSRCCGEEEIIIYSQGAd9bIy0kFMFsWZSXKzzhOwwhhGaHHS+bjKSp0tHcXdor0IUCK7NluC69WL6A7pMCCGEkOiHIhIhhBBCCCGEEBLjaERbWqgHUjC/QMwiikgkQriMSnF2cL7VtZcUxE2NzGuRpstmcbEEVq2RYG6uLhNCCCEk+qGIRAghhBBCCCGExDjqREq1e+YUFFJEIhHDcHsq949K8IhhTXUCPag6thejTaZ9h0hg9VoxN2y0lwghhBAS7VBEIoQQQgghhBBCYhyjRZq4UkNOJLOwSB0fhEQCCEZGeE8m9N6qq4hk4cpqI+6M1vaSSHDDRglu3mIvEUIIISTaoYhECCGEEEIIIYTEOBpll5yk88GiYjFLSnWekB3G4xFxVYhIhopICfZS7bjaZoorvZW9ZO2fORsZZ0cIIYTEEBSRCCGEEEIIIYSQGMeVkixGYqLOqwuptEznCdlRsF8Zid7QgsslrtSU8n2tLrjbtRUjPd1eColIZm6evUQIIYSQaIciEiGEEEIIIYQQEuskJ4mRZItIpaUSLKOIRCIEouvsnkie7juJu0dXEW/dnUhG69ZiwIlkv4aZly/B3C0igYAuE0IIISS6oYhECCGEEEIIIYTEOJXcImU+EZ8/NE/IjuLxiKd/X/EO302SjjtSEoYNEcMWhOqC4U0QV2a6GKkpumxa+6a5KVfMgkJdJoQQQkh0QxGJEEIIIYQQQgiJcVCod5weKiAFKCKRyIB9K/HQAyT1pqsk6eRjxdWxvf1I3XG1bSPujNb2krV7rs2W4MZN9hIhhBBCohmKSIQQQgghhBBCSKzjsi7vMQHEhAWCoXlCIoC7Y3tJ3H8v8fTqLoazn9UDd4d2YmRm2EvW7rl6jQSyc+wlQgghhEQzFJEIIYQQQgghhJAYxzCsy3vD0HkzENCJkGjB1amDupEcgmuzxaQTiRBCCIkJKCIRQgghhBBCCCExjmnfEhKNuLPaijsz3V4SCW7YKIFNm+0lQgghhEQzFJEIIYQQQgghhJBYBy4k24mklJVZk89eIKRpcbVsIUZGugh6d1kE8wskmLNRzJISXSaEEEJI9GKYFvY8IYQQQgghhBBCYhAU4/PvfliKXnlbJBiUxFH7SeIh+4urfZbalMziYuv/Gu7y30hIECMlRYw0a0pJFldamoh1ayRjPrVcPCDNl8IX35SiJ1+QwLr1upxy3umSeu2l4u7UQZcJIYQQEp1QRCKEEEIIIYQQQuKAopffksInXpDA2mwxkpLEaNVSXHaEmFlUpOJSg2AYYiR4VTRytUgVIxViUlroVu9L26qIhH+H5zjPFYhOqZhSRFIhPlmPu2yHVUKCuPDcFOsxjzt0H4kZSr7+XgofflZ8s+bocuLhoyTtpislYWB/XSaEEEJIdEIRiRBCCCGEEEIIiQ
```

Comment:
It is a bit odd that all of the noisy data points are above the noisefree ones.  I guess the noise realization just happened to have all positive values.  I think it would be nicer (less initial confusion at least for me) if it can be run with a different random seed, such that there are at least one negative noise realization (but perhaps to positive and two negative the best).

Reply:
changed to a different seed and now two data above and two data below the exact